pauljoegeorge/m5stickc-mrubyc-template
# [WIP] BLE Client

# To test:

- Flash BLE server to one M5StickC
  - Will start advertising ( BLE name is set to ESP_DOOR_CHIME)
- Flash BLE server to another M5StickC
  - will start searching (for ESP_DOOR_CHIME)
- BLE server & BLE client will be paired
- Click on square button (Button A) of BLE client to send chime notification
   - on BLE server you will see a message `CHIME` in blue color
- click on top button (Button B) of BLE_SERVER to disconnect / restart pairing
- click on top button (Button B) of BLE_CLIENT to disconnect / restart pairing
